test/stat_tracker_test.rb

```diff
@@ -0,0 +1,26 @@
+require './test/test_helper'
+require './lib/stat_tracker'
+
+
+
+class TestStatTracker <Minitest::Test
+
+  def test_stat_tracker_can_pull_file_locations
+    game_path = './data/games.csv'
+    team_path = './data/teams.csv'
+    game_teams_path = './data/game_teams.csv'
+    locations = {
+      games: game_path,
+      teams: team_path,
+      game_teams: game_teams_path
+    }
+
+    stat_tracker = StatTracker.from_csv(locations)
+    require "pry"; binding.pry
+
+    assert_equal 7441, stat_tracker.games.length
+    assert_equal 32, stat_tracker.teams.length
+    assert_equal 14882, stat_tracker.game_teams.length
+  end
```

Comment:
I'd like to explore adding more tests here to make sure that all the data is properly pulled in.  I'll work on this on a new branch.